src/app/shell/details/achievements/achievements.component.scss

```diff
@@ -1,4 +1,40 @@
 @import "src/app/shared/styles/card-wrapper.scss";
-.card-wrapper {
-  margin-top: 2rem;
-}
\ No newline at end of file
+@import "src/app/shared/styles/cards.scss";
+
+.mat-card {
+  margin: 2rem;
+}
+
+.wrap-for-details {
+  margin-top: 1em;
+  align-items: flex-start !important;
+}
+
+.achievement_title {
+  width: 18em;
+  font-size: 18px;
+  font-weight: 700;
+}
+.card_title {
+  font-size: 13px;
+  font-weight: 700;
+}
+
+.card_achievement {
+  min-width: 500px;
```

Comment:
let's keep it according to teh designs 
![image](https://user-images.githubusercontent.com/27493959/178214388-3fad43fc-df49-487b-b4ab-82843f053f0a.png)
![image](https://user-images.githubusercontent.com/27493959/178214415-3b3f2558-cc58-4515-8597-78733b5e73f0.png)